public class exceptionHandling {

    public static  void main(String args[]) {

        System.out.println("WE NEED TO");
        System.out.println(" ");
        System.out.println("LEARN JAVA");
        System.out.println(" ");
        System.out.println("AS QUICKLY AS POSSIBLE");
    }
//        int result = 0;
//        try {
//            Scanner sc = new Scanner(System.in);
//            System.out.println("enter the value for n1");
//            String y1 = sc.next();
//            System.out.println("enter the value for n2");
//            String y2 = sc.next();
//            int n1 = Integer.parseInt(y1);
//            int n2 = Integer.parseInt(y2);
//            System.out.println("we got two numbers.");
//            result = n1 / n2;
//        } catch (NumberFormatException e ) {
//           // System.out.println("error occureed");
//            System.out.println(e.getMessage());
//        }
//        catch(ArithmeticException e){
//            System.out.println(e.getMessage());
//        }
//        System.out.println(result);
//
//
//    }

}
